src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
+
+Page filters enable you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
+
+  ![Page Filters](search-page-filters.png)
+
+These filters are normally only persistent within the current page, but you can choose to "lock" the current filter selection such that any changes made to the "locked" filter will be persisted throughout the application and even remembered between sessions.
+
+  ![Page Filters Locking](search-page-filters-lock.png)
+
 ## Syntax
```

Comment:
I think it'll be helpful to make this more descriptive in light of the fact that we're adding content about page-level filters. 
```suggestion
## Query Syntax {#syntax}
```